boards/pba-d-01-kw2x/include/board.h

```diff
@@ -85,6 +85,14 @@ extern "C"
 #define KW2XRF_SHARED_SPI          (0)
 /** @}*/
 
+/**
```

Comment:
unrelated?

Reply:
Huh, yes ! I have no idea where does this come from...